dashboard/config/scripts/AP_practice_Qs_rqb_115_CSP_2_2_3.multi

```diff
@@ -0,0 +1,27 @@
+name 'AP_practice_Qs_rqb_115_CSP_2_2_3'
+wrong 'Logic Gates
+'
+wrong 'Transistors
+'
+wrong 'Processor
+'
+right 'Video Card
+'
```

Comment:
Nit: some of the exported data has extra new lines in the answer text (shouldn't affect how they're displayed though).

Reply:
Yeah, those are a lazy artifact of making answer text that has html human-readable in Levelbuilder. Nit noted.  But not fixing :)

Reply:
Retract ^^^ partially.  They are an artifact of the crappy formatting we got the questions in in the first place.  \r\n etc.